mne_bids/read.py

```diff
@@ -383,14 +394,32 @@ def read_raw_bids(bids_fname, bids_root, extra_params=None,
         # Get coordinate frames that electrode coordinates are in
         if kind == "meg":
             coord_frame = coordsystem_json['MEGCoordinateSystem']
+            coord_unit = coordsystem_json['MEGCoordinateUnits']
+        elif kind == 'eeg':
+            coord_frame = coordsystem_json['EEGCoordinateSystem']
+            coord_unit = coordsystem_json['EEGCoordinateUnits']
+
+            if coord_frame.lower() not in ['captrak', 'besa']:
+                warn("Defaulting to 'head' coordinate frame for "
+                     "EEG montage because mne-bids does not "
+                     "support {} coordinate frame yet.".format(coord_frame))
+            coord_frame = 'head'
```

Comment:
@adam2392 no guessing anywhere. A hard-fail would be problematic since it would make the entire file structure unreadable. Perhaps skip setting the montage with a warning message for the user to set the coordinate system correctly.

Reply:
Okay will skip by setting coord_frame to None then.

Reply:
I was using `head` because @sappelhoff initially suggested it, but it seems based on discussion that EEG might be a lot more complicated.